### DISCUSSION TITLE
Stop Popup from Adjusting

### DISCUSSION THREAD

**Discussion by sethdumaguin** [uber/react-map-gl#1992] [General]
Hello! Is there a way to stop the Popup component from adjusting its position when I pan around the map? I would like to keep it open as I pan around the map, but I don't want it to adjust its position when I move around. I appreciate the help.

https://user-images.githubusercontent.com/21695860/190830146-2db9199b-078f-410f-8b48-0344f53a2136.mov

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/docs/api-reference/popup#anchor

**Reply by sethdumaguin** [reply]
Hi! Thank you for your response! The video above has the `anchor` prop set to "bottom" but the `Popover` would still reposition itself when I pan around the map. Here is a codesandbox of what I have roughly have [link](https://codesandbox.io/s/react-mapgl-with-deckgl-forked-pn9eli). 

The intended behavior I'm trying to have is for the `Popover` to not move its position when pan around the map and stay at the top of the `Marker` component. Is that possible to do? Appreciate the help!

**Reply by Pessimistress** [reply]
The anchor prop is passed to mapbox-gl's [Popup](https://docs.mapbox.com/mapbox-gl-js/api/markers/#popup) under the hood. If it doesn't work you should report it to the Mapbox GL JS repo.
